_posts/2021-08-17-cockpit-251.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Cockpit 251
+author: garrett
+date: '2021-08-17'
+tags: cockpit, machines
+slug: cockpit-251
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 251 and cockpit-machines 250:
+
+
+## Certificate/smart card authentication to ssh and sudo
+
+Once you logged into Cockpit with a certificate, you likely need to switch to administative mode, or connect to remote machines through SSH. If your machine is part of a FreeIPA domain, you can now set up [constrained delegation rules](https://www.freeipa.org/page/V4/Service_Constraint_Delegation) so that the Cockpit certificate authentication gets forwarded to sudo and/or ssh. This is particularly useful if your user account does not have a password. Please see the [Client certification documentation](https://cockpit-project.org/guide/latest/cert-authentication.html) for details.
+
+## Logs: ...
+![Screenshot from 2021-08-09 13-11-59]({{ site.baseurl }}/images/251-logs-.png)
+
+## Machines: Delete shared directories
+
+Cockpit Machines can now delete existing filesystem devices.
+
+![screenshot of delete shared directories]({{ site.baseurl }}/images/251-machines-delete-shared-directories.png)
```

Comment:
Are we actually recursively deleting directories on the root filesystem? That seems dangerous. The screenshot shows the equivalent of `rm -rf /mnt`... is this intended? It also says "filesystem devices", but this isn't a device, it's a directory serving as a mount point. (A device would be something like `/dev/hda`, of course.)

Or are we removing the mount points from the machines? If so, it should be remove, not delete. This would require a hotfix and revised text and screenshot before release.